app/uk/gov/hmrc/incometaxproperty/controllers/JourneyAnswersController.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxproperty.controllers
+
+import play.api.Logging
+import play.api.libs.json.{JsError, JsSuccess, Json, Reads}
+import play.api.mvc.{Action, AnyContent, ControllerComponents, Result}
+import uk.gov.hmrc.incometaxproperty.actions.{AuthorisationRequest, AuthorisedAction}
+import uk.gov.hmrc.incometaxproperty.models.common._
+import uk.gov.hmrc.incometaxproperty.models.errors.{CannotParseJsonError, CannotReadJsonError, ServiceError}
+import uk.gov.hmrc.incometaxproperty.models.request.PropertyAbout
+import uk.gov.hmrc.incometaxproperty.services.PropertyService
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+class JourneyAnswersController @Inject()(propertyService: PropertyService,
+                                         auth: AuthorisedAction,
+                                         cc: ControllerComponents)(implicit ec: ExecutionContext) extends BackendController(cc) with Logging {
+
+
+  def savePropertyAbout(taxYear: TaxYear, businessId: BusinessId, nino: Nino): Action[AnyContent] = auth.async { implicit request =>
+
+    val ctx = JourneyContextWithNino(taxYear, businessId, request.user.getMtditid, nino).toJourneyContext(JourneyName.About)
+    val requestBody = parseBody[PropertyAbout](request)
+
+    requestBody match {
```

Comment:
nice, maybe in the coming tickets, we may move this to a separate trait, to standardise the handling of request payload.

Reply:
Yeah, thats correct